sa/CMakeLists.txt

```diff
@@ -188,6 +188,10 @@ set(COMMUNICATION_HEADERS
   ${COMMUNICATION_DIR}/communication_channel.hpp
   )
```

Comment:
What about adding ${BASE_DIR}/scope_exit.hpp to SA too?

Reply:
Done